pkg/reconciler/delivery/delivery.go

```diff
@@ -99,17 +109,53 @@ func (c *Reconciler) updateRoute(ctx context.Context, cfg *v1.Configuration) err
 	if err != nil {
 		return err
 	}
+
 	logger.Info("Applying updated Route object")
 	_, err = c.client.ServingV1().Routes(cfg.Namespace).Update(route)
 
+	// when the percentages reach 0/100, we know that we don't need to queue future events
```

Comment:
I don't see a 100% check?

Reply:
The else condition that prints "Progressive rollout completed" is only executed when Percent reaches 0. Since there are only 2 Revisions (for now) and more than 2 Revisions trigger an error, if one reaches 0, the other must reach 100.

Reply:
My apologies, on second thought, I think there should be a better way than doing the 0/100 check. I will send an update